config-windows.md

```diff
@@ -75,22 +75,31 @@ The following parameters can be specified:
     }
 ```
 
-### <a name="configWindowsNetwork" />Network
+## <a name="configWindowsNetwork" />Network
 
-`network` is an OPTIONAL configuration for the container's network usage.
+You can configure a container's networking options via the OPTIONAL `network` field of the Windows configuration.
 
 The following parameters can be specified:
 
-* **`egressBandwidth`** *(uint64, OPTIONAL)* - specified the maximum egress bandwidth in bytes per second for the container.
+* **`endpointList`** (array of strings, OPTIONAL) list of HNS endpoints that the container should connect to.
```

Comment:
HNS is Host Network Service on Windows (manages all container networking). I can add clarification.